src/components/main/NewItemModal.tsx

```diff
@@ -182,7 +182,7 @@ const NewItemModal = ({ open, handleClose }: Props): JSX.Element => {
       case ItemType.DOCUMENT:
         return (
           <>
-            <Typography variant="h6">
+            <Typography variant="h2" color="primary">
```

Comment:
I would let it untouched, or you change all the others. But in any case I think h2 is way too big.